From ee667cdfac44e5d77732b1e80188140d42ee372d Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Mon, 3 Nov 2014 07:52:22 -0500
Subject: [infiniband] iser: Add missing newlines to logging messages

Message-id: <82d6772551156318685163f87d62a7e96743c5bd.1415000301.git.avadai@redhat.com>
Patchwork-id: 98693
O-Subject: [Patch RHEL7.1 24/37] IB/iser: Add missing newlines to logging messages
Bugzilla: 1107622
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Doug Ledford <dledford@redhat.com>

From: Roi Dayan <roid@mellanox.com>

Logging messages need terminating newlines to avoid possible message
interleaving.  Add them.

Signed-off-by: Roi Dayan <roid@mellanox.com>
Signed-off-by: Joe Perches <joe@perches.com>
Signed-off-by: Or Gerlitz <ogerlitz@mellanox.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit e7eeffa4a073c9d5a4b04a64c5efb1edcb89fb11)
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/ulp/iser/iscsi_iser.c b/drivers/infiniband/ulp/iser/iscsi_iser.c
index f217488..eb79739 100644
--- a/drivers/infiniband/ulp/iser/iscsi_iser.c
+++ b/drivers/infiniband/ulp/iser/iscsi_iser.c
@@ -508,28 +508,28 @@ iscsi_iser_set_param(struct iscsi_cls_conn *cls_conn,
  case ISCSI_PARAM_HDRDGST_EN:
   sscanf(buf, "%d", &value);
   if (value) {
-   iser_err("DataDigest wasn't negotiated to None");
+   iser_err("DataDigest wasn't negotiated to None\n");
    return -EPROTO;
   }
   break;
  case ISCSI_PARAM_DATADGST_EN:
   sscanf(buf, "%d", &value);
   if (value) {
-   iser_err("DataDigest wasn't negotiated to None");
+   iser_err("DataDigest wasn't negotiated to None\n");
    return -EPROTO;
   }
   break;
  case ISCSI_PARAM_IFMARKER_EN:
   sscanf(buf, "%d", &value);
   if (value) {
-   iser_err("IFMarker wasn't negotiated to No");
+   iser_err("IFMarker wasn't negotiated to No\n");
    return -EPROTO;
   }
   break;
  case ISCSI_PARAM_OFMARKER_EN:
   sscanf(buf, "%d", &value);
   if (value) {
-   iser_err("OFMarker wasn't negotiated to No");
+   iser_err("OFMarker wasn't negotiated to No\n");
    return -EPROTO;
   }
   break;
-- 
1.7.1